File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/discoverHere/LockupEntrypoint/hereDevLockupApiService.java
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint;

import io.reactivex.rxjava3.core.Observable;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface hereDevLockupApiService {


    @GET("places/v1/places/lookup/")
    Observable<PlaceResponse> lookupPlace(@Query("app_id")String app_id,
                                          @Query("app_code")String app_code,
                                          @Query("source") String pvid,
                                          @Query("id")String id);
}

File: kmmsharedmodule/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverHere/LockupEntrypoint/hereDevLockupApiService.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint

import io.reactivex.rxjava3.core.Observable

interface hereDevLockupApiService {
    @GET("places/v1/places/lookup/")
    fun lookupPlace(@Query("app_id") app_id: String?,
                    @Query("app_code") app_code: String?,
                    @Query("source") pvid: String?,
                    @Query("id") id: String?): Observable<com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.PlaceResponse?>?
}